sale_commission_geo_assign_product_domain/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module allows to manage the following commission structure use case:
```

Comment:
This description doesn't seem correct.

Reply:
@pedrobaeza thanks, we added the correct ones now.